components/ai_chat/content/browser/ai_chat_tab_helper.cc

```diff
@@ -59,6 +63,30 @@ void AIChatTabHelper::PDFA11yInfoLoadObserver::AccessibilityEventReceived(
 
 AIChatTabHelper::PDFA11yInfoLoadObserver::~PDFA11yInfoLoadObserver() = default;
 
+// static
+void AIChatTabHelper::BindPageContentExtractorHost(
+    content::RenderFrameHost* rfh,
+    mojo::PendingAssociatedReceiver<mojom::PageContentExtractorHost> receiver) {
+  DCHECK(rfh);
```

Comment:
maybe not for this PR, but chromium's general direction is to replace most DCHECKs with CHECKs, see https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/checks.md and the style guide